web_widget_table/README.rst

```diff
@@ -0,0 +1,88 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===============
+Web Widget Table
+===============
+
+A module to add datatable https://datatables.net/ as a widget
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=============
+
+Use this widget by saying:
+
+.. code-block:: xml
+   :linenos:
+<field name="my_table" widget="widget_table" options="{
+                            'field_name': 'child_ids',
+                            'datatable_params':{'searching': False,
+                            'paging': False,
+                            'info': False},
+                            'fields': ['name','date_start', 'date_end', 'real_date_start', 'real_date_end', 'total_time'],
+                            'headers': ['Name','Date Start', 'Date End', 'Real Date Start', 'Real Date End', 'Total Time']}"/>
+
+"my_table" field must be of type char, it is only used to display the table. For example:
```

Comment:
Is it possible to make the widget work directly on one2many fields, without adding a new 'empty' field?

Reply:
Hey, thanks it makes sense what you are saying, I did it like this because the use case that I thought was to add any field of the model to the table, not only the fields of many2many record records. I created an issue for this comment, I think it would be more intuitive to if a many2many field is referenced to make it as you say, so I created an issue for this: https://github.com/OCA/web/issues/1273